modules/test/playwright/tests/export-import-web/main/site.import.spec.ts

```diff
@@ -410,11 +413,178 @@ test('can import a lar file selecting some items to import', async ({
 	).toBeVisible();
 });
 
+testWithExportImportAtInstanceLevelFF(
+	'Can export and import a site with all the exportable items and created with the test site initializer',
+	{tag: '@LPD-64056'},
+	async ({apiHelpers, exportImportPage, systemSettingsPage}) => {
+		test.setTimeout(180000);
+
+		let exportFilePath: string;
+		let exportName: string;
+		let exportableItems1: Map<string, number>;
+		let exportableItems2: Map<string, number>;
+		let exportableItems3: Map<string, number>;
+		let originalOverallMaximumUploadRequestSize: string;
+		let site1;
+		let site2;
+
+		await test.step('Increase the maximum upload request size', async () => {
+			await systemSettingsPage.goToSystemSetting(
+				'Infrastructure',
+				'Upload Servlet Request'
+			);
+
+			originalOverallMaximumUploadRequestSize =
+				await systemSettingsPage.page
+					.getByLabel('Overall Maximum Upload Request Size')
+					.textContent();
+
+			await systemSettingsPage.page
+				.getByLabel('Overall Maximum Upload Request Size')
+				.fill('200000000');
+
+			await systemSettingsPage.saveButton.click();
+		});
+
+		try {
+			await test.step('Create the site 1 from the template', async () => {
+				site1 = await apiHelpers.headlessSite.createSite({
+					name: getRandomString(),
+					templateKey:
+						'com.liferay.exportimport.test.site.initializer',
+					templateType: 'site-initializer',
+				});
+
+				apiHelpers.data.push({id: site1.id, type: 'site'});
+			});
+
+			await test.step('Export the site 1', async () => {
+				await exportImportPage.goToExport(site1.friendlyUrlPath);
+
+				exportableItems1 = await exportImportPage.getExportableItems();
+
+				expect(exportableItems1.size).toBeGreaterThan(0);
+
+				exportName = `MyExport-${getRandomString()}`;
+
+				await exportImportPage.exportAll(exportName);
+
+				await expect(
+					exportImportPage.page
+						.locator(
+							'//h2[span[normalize-space()="' + exportName + '"]]'
+						)
+						.first()
+						.locator('../..')
+						.getByText('Successful')
+				).toBeVisible({timeout: 60000});
```

Comment:
Did you consider the locator I added in another commit? They are better for the long term